### PR TITLE
Enable manual usage of locally running wire-server 

### PR DESCRIPTION
**Pull request by akshaymankar**
Webapp repository needed a little change for this to work: https://github.com/wireapp/wire-webapp/pull/17844

## Checklist

 - [x] Add a new entry in an appropriate subdirectory of `changelog.d`
 - [x] Read and follow the [PR guidelines](https://docs.wire.com/developer/developer/pr-guidelines.html)

### CODE REVIEW

File: integration/test/Testlib/RunServices.hs [hunk 1]
```diff
@@ -57,6 +77,36 @@ main = do
...
+            )
             [backendA, backendB]
         liftIO run
+
+manualTestingOverrides :: ServiceOverrides
+manualTestingOverrides =
+  let smtpEndpoint = object ["host" .= "localhost", "port" .= (2500 :: Int)]
+      authSettings =
+        object
+          [ "userTokenTimeout" .= (4838400 :: Int),
```
Comment (supersven) [inline integration/test/Testlib/RunServices.hs:93]: I'm usually too lazy to calculate and would write a `let` or `where` binding for such numbers. E.g. `fiftysixDays = 4838400`.

But, that's probably a matter of taste :smile:

**Review by supersven — APPROVED** [review]
Some minor nitpicks. Other than that: LGTM :+1:

File: docs/src/developer/developer/building.md [hunk 2]
```diff
@@ -218,3 +218,26 @@ After all containers are up you can use these Makefile targets to run the tests
    ```
 
    `TASTY_NUM_THREADS` can also be set to other values, it defaults to number of cores available.
+
+## How to run the webapp locally against locally running backend
+
+1. Clone the webapp from: https://github.com/wireapp/wire-webapp
+2. Install these depedencies needed for the webapp:
```
[the remark this answers is not in the document]
Comment (akshaymankar) [reply docs/src/developer/developer/building.md:225]: Yes, I was thinking, but then I thought clearly noone would maintain that. So, its probably fine to just `nix shell nixpkgs#nodejs ...`

File: integration/test/Testlib/RunServices.hs (hunk 1 above)
Comment (akshaymankar) [reply integration/test/Testlib/RunServices.hs:93]: I copy pasted these from default values of the helm charts and I have no clue what they correspond to, I'd wager noone would care about this until they testing something specific at which point they'd probably just change the values temporarily.

**Pull request merged**